development/mimir-microservices-mode/compose-up.sh

```diff
@@ -7,12 +7,13 @@
 set -e
 
 SCRIPT_DIR=$(cd "$(dirname "$0")" && pwd)
+ALPINE_VERSION=$(make -s -f ../../Makefile print-alpine-version)
```

Comment:
```suggestion
ALPINE_VERSION=$(make -s -f ${SCRIPT_DIR}/../../Makefile print-alpine-version)
```

Same for other `compose-up.sh` scripts.